SCons/Script/Main.xml

```diff
@@ -943,6 +944,51 @@ Example:
 SetOption('max_drift', 0)
 </example_commands>
 </summary>
+
+
 </scons_function>
 
+    <scons_function name="ValidateOptions">
+        <arguments signature="global">
+            ([throw_exception=False])
+        </arguments>
+
+        <summary>
+            <para>
+                Check that all the options specified on the command line are either defined by SCons itself
+                or defined by &f-link-AddOption;.
+            </para>
+            <para>
+                If <parameter>throw_exception</parameter> is True, &f-ValidateOptions; will raise a
```

Comment:
move this below the next para for better flow. To be pedantic, add the word optional ("If optional throw_exception...."). Can leave the function name out here, we know what this refers to. Maybe add a line about then being able to catch the exception and (do something).  Use `<exceptionname>` for the markup.

Reply:
Should be addressed in next push.

Reply:
I'll revise this slightly: I'd say the first para "Check that all the..." should have the third para jointed into it "If there are any command line options...".  That's all part of the initial description.